### PR TITLE
Feat: Card

### PR DESCRIPTION
**Pull request by dikshadhote**
Following cards have been added 
1.Card with only text
image

2.horizontal card
image

3.Vertical Card
image

4. Card with shadow

image

5.Card with badge
image

6.Card with dismiss
image

7.Card with Overlay
image

preview link:
https://dreamui-card.netlify.app/documents/documents.html#card

### CODE REVIEW

**Comment by psychoactiv** [comment]
Code looks good to merge.

**Review by psychoactiv — APPROVED** [review]

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+.card-img-hori{
+    width: 40%;
+    height: 100%;
+}
+
+.card-img-hori .img-hori{
+    border-radius: 5px 0px 0px 5px;
+}
+
+
```
Comment (mariyasada) [inline components/card.css]: remove unnecessary space

File: utility.css [hunk 2]
```diff
@@ -298,13 +298,59 @@ background-color: var(--component-secondary-dark);
...
-} 
\ No newline at end of file
+} 
+
+.flex-justify-space-between{
+  justify-content: space-between;
+}
+
+
+
```
Comment (mariyasada) [inline utility.css]: remove this space

**Review by mariyasada — APPROVED** [review]
remove unnecessary space

**Pull request merged**